frontend/src/components/ApplicationConsent.js

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable react/jsx-indent */
+import React, { useState } from 'react';
+import Checkbox from '@mui/material/Checkbox';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Box from '@mui/material/Box';
+
+const ApplicationConsent = (props) => {
+  const commonStyles = {
+    bgcolor: 'background.paper',
+    width: '100%',
+    height: 500,
+    overflowY: 'scroll'
+  };
+  const emliAddress = (
+    <div className="emli-address">
```

Comment:
Let's use the <address> tag
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address